holo-isis/src/packet/tlv.rs

```diff
@@ -1303,6 +1315,37 @@ impl IpReachTlvEntry for ExtIpv4Reach {
     }
 }
 
+// ===== impl BierInfoTlv =====
+
+impl BierInfoTlv {
+    pub(crate) fn decode(tlv_len: u8, buf: &mut Bytes) -> DecodeResult<Self> {
+        if tlv_len < 5 {
```

Comment:
Please define a constant for this, like `Self::MIN_LEN`.

Reply:
Fixed in 3568865.